x-pack/test/security_solution_endpoint_api_int/apis/metadata.ts

```diff
@@ -222,7 +221,7 @@ export default function ({ getService }: FtrProviderContext) {
           (ip: string) => ip === targetEndpointIp
         );
         expect(resultIp).to.eql([targetEndpointIp]);
-        expect(body.hosts[0].metadata.event.created).to.eql(1618841405309);
+        expect(body.hosts[0].metadata.event.created).to.eql(1626897841950);
```

Comment:
I'm wondering if this will fail again in the future. I think we should mock the date to ensure it's always the same

Reply:
for sure. created a follow up tech debt ticket to resolve this.